src/models/StrategyPrepSchema.ts

```diff
@@ -0,0 +1,84 @@
+import { model, Schema } from 'mongoose';
+
+import { createLogger } from '../logger';
+import { IStrategyPrep } from './Models';
+
+const logger = createLogger('StrategyPrepSchema');
```

Comment:
i didnt see the usage of this logger on this file. maybe a logger.debug('StrategyPrepSchema') is good?

Reply:
it's just boilerplate for when it's needed. all other models have it. it's to remind myself that if I want to log something, I need to use this logger